### ISSUE TITLE
How to create abilities from tinker?

### ISSUE THREAD

**Issue by pintend** [JosephSilber/bouncer#591]
running ```Bouncer::allow('admin')->to('ban-users');``` in tinker returns ```Silber\Bouncer\Conductors\Lazy\ConductsAbilities```
and never saves in the database

how do i save a lazy ability?

**Comment by JosephSilber** [comment]
Yeah. That's a problem. It's actually a problem with any lazy object. Laravel is full of those.

I've been thinking of maybe adding a `save` method, which would allow you to manually commit it. It would also have to save that state on the object, so that it doesn't run again when destructuring.

It's bad either way. I guess that's the tradeoff for using lazy objects.

**Comment by krismanning** [comment]
I've just come across this, which is weird because its been working for ages, then suddeny I cant add abilities from Tinker.
Was there any update on this or a workaround?

many thanks.

**Comment by pintend** [comment]
You can call `->_destruct()` manually
